docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst

```diff
@@ -251,9 +252,9 @@ Manual deployment
 ^^^^^^^^^^^^^^^^^
 
 If you want to manually deploy ``Sequencer.sol`` to your Ethereum network, the file
-``<<canton-release/examples/e03-ethereum-sequencer/qbft-testnet/Sequencer-binary``
-contains the compiled Solidity code you need to deploy. For Besu, for example, you will need to specify the
-contents of ``Sequencer-binary`` in ``"code": "..."`` as documented `here <https://besu.hyperledger.org/en/stable/HowTo/Configure/Contracts-in-Genesis/#pre-deploying-contracts-in-the-genesis-file>`_.
+``<<canton-drivers-release>>/examples/e03-ethereum-sequencer/qbft-testnet/Sequencer.bin-runtime``
+contains the compiled Solidity code you need to deploy. For Besu, for example, you will need to specify the contents of ``Sequencer.bin-runtime`` in ``"code": "..."`` as documented `here <https://besu.hyperledger.org/stable/private-networks/how-to/configure/contracts>`_.
+This can also be seen in the ``generate-testnet.sh``. When code is added to the genesis block, the constructor is not used to initiate, so we need to do values instanced with ``"storage": "..."``.
```

Comment:
This sentence is not clear to me. I trip over "is not used to initiate" and "we need to do values"

Reply:
Agree that the last sentence could be clearer. I don't know enough to edit it.

Reply:
does the new wording sound better to you? tried to update it a bit

Reply:
Much clearer, thank you!